File: web/src/model/victron_controller/dashboard/from_0_1_0_wsservermessage_log.ts
// @ts-nocheck
import {Log as v0_1_0_Log} from './v0_1_0/WsServerMessage'
import {Log as dashboard_Log} from './WsServerMessage'

export function convert__ws_server_message__log__from__0_1_0(from: v0_1_0_Log): dashboard_Log {
    return new dashboard_Log (
        JSON.parse(JSON.stringify(from.body))
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/v0_2_0/from_0_1_0_wsservermessage_log.rs


pub fn convert__wsservermessage__log__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::ws_server_message::Log) -> crate::victron_controller::dashboard::v0_2_0::ws_server_message::Log {
    crate::victron_controller::dashboard::v0_2_0::ws_server_message::Log {
        body: serde_json::from_value(serde_json::to_value(&from.body).unwrap()).unwrap(),
    }
}